trello-history/trello-history.md

```diff
@@ -0,0 +1,212 @@
+# How we extended the trello experience with power-ups
```

Comment:
The "how" written like this, is a bit accuser... You can just say that we (us and client) can lost informations sometimes when editing the description?

Reply:
The how here stands for "comment" not "why" :) But I agree with you on that point, I have to work a little bit more on that intro to get rid of the accusing part